tailwind.config.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
         mainLight: '#A5A4BF',
         mainActive: '#A3A0FB',
         secondary: '#A6A7AD',
+        active: '#3A87FF',
+        inactive: '#EAEAEA',
+        app: '#F0F0F7',
+        text: '#4E4F5C',
+        tableHeader: '#F5F6FA',
+        tableHeaderText: '#A3A6B4',
         active: '#0ea5e9',
```

Comment:
I've made some modifications in the way colors are organised within theme in tailwind.config within PR #16 . Just mentioning it here